src/rubintv/handlers/external/endpoints.py

```diff
@@ -261,17 +267,40 @@ async def get_historical_day_data(request: web.Request) -> web.Response:
     the_date = date(year, month, day)
     day_dict = historical.get_events_for_date(camera, the_date)
     day_events = flatten_events_dict_into_list(camera, day_dict)
-    grid_columns = build_grid_columns_css(camera.channels)
+
+    metadata_json = get_metadata_json(bucket.name, camera.slug, date_str)
+
     page = get_formatted_page(
         "cameras/data-table-header-with-day-channels.jinja",
         camera=camera,
         date=the_date,
         events=day_events,
-        grid_columns=grid_columns,
+        metadata=metadata_json,
     )
     return web.Response(text=page, content_type="text/html")
 
 
+def get_metadata_json(
+    bucket_name: str, camera_slug: str, date_str: str
+) -> str:
+    metadata_json = "{}"
+    metadata_url = get_metadata_url(bucket_name, camera_slug, date_str)
+    metadata_json = requests.get(metadata_url).text
```

Comment:
Sorry, comment from before went somewhere very strange.
I think this is a mistake, right? You're accessing the result before the check for 200 return code.